pytorch_lightning/trainer/trainer.py

```diff
@@ -781,6 +782,135 @@ def slurm_job_id(self) -> int:
             job_id = None
         return job_id
 
+    @staticmethod
+    def add_default_args(parent_parser):
```

Comment:
maybe rename to add_trainer_args? and ideally it can be called by:
```python
import pytorch_lightning as pl
from argparse import ArgumentParser

parser = ArgumentParser()
parser = pl.add_trainer_args(parser)
```